HollowKnightSplitSettings.cs

```diff
@@ -687,6 +691,8 @@ public enum SplitName {
         EnterSanctum,
         [Description("Soul Sanctum w/ Shade Soul (Transition)"), ToolTip("Splits when entering Soul Sanctum after obtaining Shade Soul")]
         EnterSanctumWithShadeSoul,
+        [Description("Tower of Love (Transition)"), ToolTip("Splits when entering the Tower of Love")]
+        TowerOfLove,

```

Comment:
Whichever implementation in `HollowKnightComponent.cs` you go with, I'd like the name to start with `Enter`.
```suggestion
        EnterLoveTower,
```